### PR TITLE
Broken links removed + fixed some 301 errors

### PR DESCRIPTION
**Pull request by vlio20**
Note that I added a rename-redirect.js script which rename redirects.

### CODE REVIEW

**Comment by vlio20** [comment]
@rolandjitsu, As you can see all broken (404) links were removed + all 301 redirects were solved as well.  
I am not sure how should the 302 and the dups be solved.  

Also please note that I added rename-redirect.js script which will replace links that have redirect error (301).  

It would be nice if you would merge this PR as it solves most of the errors.

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+  <a href="undefined" target="_blank">
     <img src="/media/awesome-angular2.png" alt="Awesome Angular 2" />
   </a>
 </p>
 
-# Awesome Angular 2 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Build Status](https://travis-ci.org/AngularClass/awesome-angular2.svg?branch=master)](https://travis-ci.org/AngularClass/awesome-angular2)
+# Awesome Angular 2 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Build Status](https://api.travis-ci.org/AngularClass/awesome-angular2.svg?branch=master)](https://travis-ci.org/AngularClass/awesome-angular2)
 
-Awesome list of [Angular 2](https://angular.io/) seed repos, starters, boilerplates, examples, tutorials, components, modules, videos, and anything else in the Angular 2 ecosystem
+Awesome list of [Angular 2](undefined) seed repos, starters, boilerplates, examples, tutorials, components, modules, videos, and anything else in the Angular 2 ecosystem
```
Comment (rolandjitsu) [inline README.md]: This does not seem right.

**Comment by vlio20** [comment]
Oh I didn't see it (reminder not to code late). Do you want me to rollback to the commit which removes all broken links (404). I will exclude the script.

File: README.md [hunk 2]
```diff
@@ -93,10 +93,10 @@ Table of contents:
 > Angular is a development platform for building mobile and desktop web applications.
 
 #### Official Resources
-* [Site](https://angular.io/)
-* [Blog](http://angularjs.blogspot.com/)
-* [Documentation](https://angular.io/docs/js/latest/)
-* [Cheatsheet](https://angular.io/cheatsheet)
+* [Site](undefined)
```
Comment (rolandjitsu) [inline README.md]: Again, `undefined` does not seem right.

**Comment by rolandjitsu** [comment]
@vlio20 you can just submit your own link. I am taking care of some of the links right now. You could leave the removed 404s though so I don't have to go through them again.

File: README.md [hunk 3]
```diff
@@ -93,10 +93,10 @@ Table of contents:
...
+* [Blog](http://angularjs.blogspot.co.il/)
```
Comment (rolandjitsu) [inline README.md]: I think the url should be `.com`, you get the redirected depending on the locale anyway, we just need to add it to the urls to be ignored when tested.

File: README.md [hunk 4]
```diff
@@ -93,10 +93,10 @@ Table of contents:
...
+* [Documentation](undefineddocs/js/latest/)
```
Comment (rolandjitsu) [inline README.md]: Note the `undefined`.

File: README.md [hunk 5]
```diff
@@ -93,10 +93,10 @@ Table of contents:
...
+* [Cheatsheet](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined` again.

File: README.md [hunk 6]
```diff
@@ -106,15 +106,15 @@ Table of contents:
...
-* [/r/Angular2 Subreddit](https://www.reddit.com/r/angular2/)
+* [/r/Angular2 Subreddit](https://www.reddit.com/r/Angular2/)
 * [Angular Group on Facebook](https://www.facebook.com/groups/angular2/)
 * [AngularJS on Google+](https://plus.google.com/+AngularJS/posts)
 * [AngularAir](http://angular-air.com/) podcast and live broadcast
-* [Adventures in Angular](https://devchat.tv/adventures-in-angular) podcast
+* [Adventures in Angular](https://devchat.tv/adv-in-angular) podcast
 * [ng-newsletter](http://www.ng-newsletter.com) email newsletter
-* [tryangular2](http://www.tryangular2.com/)
+* [tryangular2](http://ww1.tryangular2.com)
```
Comment (rolandjitsu) [inline README.md]: This can be removed, it seems not to be valid anymore.

File: README.md [hunk 7]
```diff
@@ -106,15 +106,15 @@ Table of contents:
...
 * [Built With Angular 2](http://builtwithangular2.com/)
-* [`#angular2 Slack Channel`](https://dartlang.slack.com/messages/angular2) on dartlang.slack.com ([Sign up](https://dartlang-slack.herokuapp.com/))
+* [`#angular2 Slack Channel`](https://dartlang.slack.com/?redir=%2Fmessages%2Fangular2) on dartlang.slack.com ([Sign up](undefined))
```
Comment (rolandjitsu) [inline README.md]: Note the `undefined` again.

File: README.md [hunk 8]
```diff
@@ -161,23 +161,23 @@ Table of contents:
 * [Ciro Nunes](https://twitter.com/cironunesdev/)
 * [Maxim Salnikov](https://twitter.com/webmaxru)
 * [Deborah Kurata](https://twitter.com/deborahkurata)
-* [**{{** add_expert **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_expert **}}**](undefined)
```
Comment (rolandjitsu) [inline README.md]: This should be left alone and added to the list of ignored urls.

File: README.md [hunk 9]
```diff
@@ -161,23 +161,23 @@ Table of contents:
...
 
 #### Server-Side Rendering
-* [Angular 2 Universal Repository (GitHub)](https://github.com/angular/universal)
+* [Angular 2 Universal Repository (GitHub)](undefined)
```
Comment (rolandjitsu) [inline README.md]: We have `undefined` again.

File: README.md [hunk 10]
```diff
@@ -161,23 +161,23 @@ Table of contents:
...
 
 #### Material Design
-* [Official Angular 2 Material Design (GitHub)](https://github.com/angular/material2)
+* [Official Angular 2 Material Design (GitHub)](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined` should not be here.

File: README.md [hunk 11]
```diff
@@ -161,23 +161,23 @@ Table of contents:
...
 * [md2](https://www.npmjs.com/package/md2) Angular2 based Material Design components, directives and services are Accordion, Autocomplete, Collapse, Colorpicker, Datepicker, Dialog(Modal), Menu, Multiselect, Select, Switch, Tabs, Tags(Chips), Toast and Tooltip.
 * [ng2-material](https://www.npmjs.com/package/ng2-material) set of material2 examples and extra components (data table, dialog, ...) built on top of @angular2-material/core
 * [angular2-mdl](https://www.npmjs.com/package/angular2-mdl) Angular 2 components, directives and styles based on Material Design Lite
 * [mdl-ext](https://www.npmjs.com/package/mdl-ext) Material Design Lite Ext (carusel, grid, lightbox, selectfield, sticky header, ...)
 
 #### Cheatsheet
-* [Official Angular 2 Cheatsheet](https://angular.io/cheatsheet)
+* [Official Angular 2 Cheatsheet](undefined)
```
Comment (rolandjitsu) [inline README.md]: Same as above.

File: README.md [hunk 12]
```diff
@@ -229,7 +229,7 @@ Http is available as an injectable class, with methods to perform http requests.
 ##### Test
 * [Testing Http services in Angular 2 with Jasmine](http://chariotsolutions.com/blog/post/testing-http-services-angular-2-jasmine/)
 * [Testing UI Components with the TestComponentBuilder](http://chariotsolutions.com/blog/post/testing-angular-2-components-unit-tests-testcomponentbuilder/)
-[**{{** help_out **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+[**{{** help_out **}}**](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined` again.

**Comment by rolandjitsu** [comment]
Just make sure to get latests in a few minutes as I am pushing some changes as well.

File: README.md [hunk 13]
```diff
@@ -240,10 +240,10 @@ Http is available as an injectable class, with methods to perform http requests.
 * [Official web page](https://webpack.github.io/)
 * [Angular 2 Webpack Starter from AngularClass](https://github.com/AngularClass/angular2-webpack-starter)
 * [Angular 2 Webpack from preboot](https://github.com/preboot/angular2-webpack)
-* [Angular 2 Nightly Webpack Starter](https://github.com/qdouble/angular2webpack2-starter)
+* [Angular 2 Nightly Webpack Starter](https://github.com/qdouble/angular-webpack2-starter)
 * [Angular 2 Webpack with Visual Studio ASP.NET Core from Damien Bowden](https://github.com/damienbod/Angular2WebpackVisualStudio)
 * [Angular 2 Typescript Webpack from schempy](https://github.com/schempy/angular2-typescript-webpack)
-* [Angular 2 Webpack  Tour of Heroes from michaelbazos](https://github.com/michaelbazos/angular2-starter) - Tour of Heroes official typescript tutorial built with webpack
+* [Angular 2 Webpack  Tour of Heroes from michaelbazos](undefined) - Tour of Heroes official typescript tutorial built with webpack
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 14]
```diff
@@ -253,18 +253,18 @@ Http is available as an injectable class, with methods to perform http requests.
...
-    * [Using Http](https://auth0.com/blog/2015/10/15/angular-2-series-part-3-using-http/)
-    * [Create a Desktop App with Angular 2 and Electron](https://auth0.com/blog/2015/12/15/create-a-desktop-app-with-angular-2-and-electron/)
+    * [Working with Pipes](https://auth0.com/blog/angular2-series-working-with-pipes/)
+    * [Domain Models and Dependency Injection](https://auth0.com/blog/angular-2-series-part-2-domain-models-and-dependency-injection/)
+    * [Using Http](https://auth0.com/blog/angular-2-series-part-3-using-http/)
+    * [Create a Desktop App with Angular 2 and Electron](https://auth0.com/blog/create-a-desktop-app-with-angular-2-and-electron/)
 * thoughtram
   * Angular 2 Template Syntax Demystified
-    * [Part 1](http://blog.thoughtram.io/angular/2015/08/11/angular-2-template-syntax-demystified-part-1.html)
+    * [Part 1](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined` again.

File: README.md [hunk 15]
```diff
@@ -291,7 +291,7 @@ Http is available as an injectable class, with methods to perform http requests.
 * [Creating realtime apps with Angular 2 and Meteor - Uri Goldshtein](https://www.youtube.com/watch?v=3FT0BqYASCo)
 
 #### Books
-* [ng-book 2](https://www.ng-book.com/2/) `fullstack.io`
+* [ng-book 2](undefined) `fullstack.io`
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 16]
```diff
@@ -301,29 +301,29 @@ Http is available as an injectable class, with methods to perform http requests.
 * [Rangle's Angular 2 training](https://www.gitbook.com/book/rangle-io/ngcourse2/details) `Rangle.io`
 
 #### On-Site Training
-* [AngularClass](https://angularclass.com)
-* [Angular Boot Camp](http://angularbootcamp.com)
+* [AngularClass](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 17]
```diff
@@ -301,29 +301,29 @@ Http is available as an injectable class, with methods to perform http requests.
...
+* [Angular Boot Camp](https://angularbootcamp.com/)
 * [thoughtram](http://thoughtram.io/training.html)
 * [Egghead.io](https://egghead.io/angularjs-enterprise-training-workshop)
-* [ng-book](https://www.ng-book.com/2/)
+* [ng-book](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 18]
```diff
@@ -301,29 +301,29 @@ Http is available as an injectable class, with methods to perform http requests.
...
+  * [The Core Concepts of Angular 2](https://medium.com/m/global-identity?redirectUrl=https://vsavkin.com/the-core-concepts-of-angular-2-c3d6cbe04d04)
+  * [Two Phases of Angular 2 Applications](https://medium.com/m/global-identity?redirectUrl=https://vsavkin.com/two-phases-of-angular-2-applications-fda2517604be)
+  * [Change Detection in Angular 2](https://medium.com/m/global-identity?redirectUrl=https://vsavkin.com/change-detection-in-angular-2-4f216b855d4c)
+  * [Better Support For Functional Programming In](http://vsavkin.tumblr.com/post/108837493941/better-support-for-functional-programming-in#_=_)
+  * [Angular 2 Bits Unified Dependency Injection](http://vsavkin.tumblr.com/post/102965317996/angular-2-bits-unified-dependency-injection#_=_)
+  * [Angular 2 Router]()
 
 * AngularClass
-  * [Automated Angular 2 Conventions with Webpack](https://angularclass.com/blog/automated-angular-2-conventions-with-webpack/)
+  * [Automated Angular 2 Conventions with Webpack](undefined/blog/automated-angular-2-conventions-with-webpack/)
```
Comment (rolandjitsu) [inline README.md]: One of those `undefined` again.

File: README.md [hunk 19]
```diff
@@ -336,7 +336,7 @@ Http is available as an injectable class, with methods to perform http requests.
   * [Dependency Injection in Angular 2](http://blog.thoughtram.io/angular/2015/05/18/dependency-injection-in-angular-2.html)
   * [Routing in Angular 2 **Depreciated**](http://blog.thoughtram.io/angular/2015/06/16/routing-in-angular-2.html)
   * [Routing in Angular 2 Revisited](http://blog.thoughtram.io/angular/2016/06/14/routing-in-angular-2-revisited.html)
-  * [Angular 2 Template Syntax Demystified - Part 1](http://blog.thoughtram.io/angular/2015/08/11/angular-2-template-syntax-demystified-part-1.html)
+  * [Angular 2 Template Syntax Demystified - Part 1](undefined)
```
Comment (rolandjitsu) [inline README.md]: One more `undefined`

File: README.md [hunk 20]
```diff
@@ -347,13 +347,13 @@ Http is available as an injectable class, with methods to perform http requests.
   * [Debugging Angular 2 Applications](http://tutorials.pluralsight.com/front-end-javascript/debugging-angular-2-applications)
 
 #### Integrations
-* [FalcorJS + Angular2 (Video)](https://youtu.be/z8UgDZ4rXBU)
-* [Angular2-Meteor](http://www.angular-meteor.com/angular2)
+* [FalcorJS + Angular2 (Video)](https://www.youtube.com/watch?v=z8UgDZ4rXBU&feature=youtu.be)
+* [Angular2-Meteor](https://www.angular-meteor.com/angular2)
 * [nativescript-angular](https://github.com/NativeScript/nativescript-angular)
-* [react-native-renderer](https://github.com/angular/react-native-renderer)
+* [react-native-renderer](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 21]
```diff
@@ -347,13 +347,13 @@ Http is available as an injectable class, with methods to perform http requests.
...
 
 #### Components
-* [Material 2](https://github.com/angular/material2) - Angular team's Material Design components built on top of Angular 2
+* [Material 2](undefined) - Angular team's Material Design components built on top of Angular 2
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 22]
```diff
@@ -366,20 +366,20 @@ Http is available as an injectable class, with methods to perform http requests.
...
 * [angular2-google-maps](https://github.com/SebastianM/angular2-google-maps) Angular2 directives for Google Maps
-* [ng2-radio-group](https://github.com/pleerock/ng2-radio-group) Angular2 directives for radio and checkbox inputs and radio input groups
+* [ng2-radio-group](https://github.com/pleerock/ng2-select-controls) Angular2 directives for radio and checkbox inputs and radio input groups
 * [ng2-dropdown](https://github.com/pleerock/ng2-dropdown) Dropdown menu for angular2 and bootstrap 3
 * [ng2-progress-bar](https://github.com/pleerock/ng2-progress-bar) Simple progress bar for angular2 and bootstrap 3
 * [ng2-tabs](https://github.com/pleerock/ng2-tabs) Tabs control for angular2 and bootstrap 3
 * [ng2-modal](https://github.com/pleerock/ng2-modal) Modal window for angular2 and bootstrap 3
 * [ng2-paginator](https://github.com/pleerock/ng2-paginator) Pagination control for angular2 and bootstrap 3
-* [fuel-ui](https://github.com/FuelInteractive/fuel-ui) Bootstrap 4 components and directives for use in Angular 2
+* [fuel-ui](undefined) Bootstrap 4 components and directives for use in Angular 2
```
Comment (rolandjitsu) [inline README.md]: One more `undefined`.

File: README.md [hunk 23]
```diff
@@ -398,18 +398,18 @@ Http is available as an injectable class, with methods to perform http requests.
 * [Wijmo 5](http://wijmo.com/products/wijmo-5/) - Set of UI components for Angular2
 * [ng2d3](https://github.com/swimlane/ng2d3) - Native Angular2 Chart Rendering Framework with D3 for utils
 * [ng2-swipeable-card](https://github.com/JayKan/ng2-swipeable-card) - Swipeable card component for Angular2 
-* [ng2-date-picker](https://vlio20.github.io/ng2-date-picker) - Highly configurable date picker built for Angular 2
+* [ng2-date-picker](https://vlio20.github.io/ng2-date-picker////) - Highly configurable date picker built for Angular 2
```
Comment (rolandjitsu) [inline README.md]: Typo?

File: README.md [hunk 24]
```diff
@@ -398,18 +398,18 @@ Http is available as an injectable class, with methods to perform http requests.
...
 
 #### Pipes
-* [fuel-ui](https://github.com/FuelInteractive/fuel-ui) OrderBy and Range pipes ported from Angular 1.x to Angular 2
+* [fuel-ui](undefined) OrderBy and Range pipes ported from Angular 1.x to Angular 2
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 25]
```diff
@@ -425,10 +425,10 @@ Http is available as an injectable class, with methods to perform http requests.
 > Universal (isomorphic) javascript support for Angular 2
 
 #### Universal General Resources
-* [Universal Angular 2 Repository (GitHub)](https://github.com/angular/universal)
+* [Universal Angular 2 Repository (GitHub)](undefined)
```
Comment (rolandjitsu) [inline README.md]: `undefined` and one more below.

File: README.md [hunk 26]
```diff
@@ -438,9 +438,9 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 
 #### TypeScript General Resources
 * [TypeScript](http://www.typescriptlang.org/) Official Website for TypeScript
-* [REPL](http://www.typescriptlang.org/Playground) Official TypeScript REPL that runs entirely in your browser
+* [REPL](http://www.typescriptlang.org/Playground///) Official TypeScript REPL that runs entirely in your browser
```
Comment (rolandjitsu) [inline README.md]: Typo? Should probably be `http://www.typescriptlang.org/play/`

File: README.md [hunk 27]
```diff
@@ -438,9 +438,9 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 * [TypeScript Repository (GitHub)](https://github.com/Microsoft/TypeScript) Official GitHub Repo for TypeScript
-* [TSD](http://definitelytyped.org/tsd/) TypeScript Definition manager for DefinitelyTyped
+* [TSD](http://angular-meteor.com/angular2) TypeScript Definition manager for DefinitelyTyped
```
Comment (rolandjitsu) [inline README.md]: This does not seem right, it redirects to `tsdpm.org` which seems to be obsolete.

File: README.md [hunk 28]
```diff
@@ -459,16 +459,16 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
+* [ng2Boilerplate](https://github.com/born2net/Angular-kitchen-sink) - The ultimate Angular 2 bootstrap single page app - touches on all key and advanced aspects of Angular2.
 * [angular2-demo](https://github.com/amcdnl/angular2-demo) Minimal Angular2 Demo using TypeScript / Sass / Npm Tasks
 * [Angular 2 Dashboard Starter](https://github.com/hasanhameed07/angular2-dashboard-starter) - Ready to use dashboard control panel seed project based on Angular 2 and AdminLTE bootstrap theme.
 * [ng2-admin](https://github.com/akveo/ng2-admin) - Beautiful admin panel framework based on Angular 2, Bootstrap 4 and Webpack
 * [Angular 2 quickstart seed](https://github.com/valor-software/angular2-quickstart)
 * [Angular 2 full code coverage](https://github.com/danday74/angular2-coverage) - Solid tested SystemJS and gulp workflow ready for your code using Angular2 final release (2.1.0) .. Demonstrates unit testing and full code coverage
-* [Angular 2 webpack](https://github.com/michaelbazos/angular2-starter) - Tour of Heroes official typescript tutorial built with webpack
+* [Angular 2 webpack](undefined) - Tour of Heroes official typescript tutorial built with webpack
```
Comment (rolandjitsu) [inline README.md]: Yet another `undefined`.

File: README.md [hunk 29]
```diff
@@ -459,16 +459,16 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 * [ng2-boiler](https://github.com/amcdnl/ng2-boiler) - A bare-bones simple starter with Angular2, TypeScript and Webpack configured.
 * [Angular Webpack Starter](https://github.com/antonybudianto/angular-webpack-starter) - Angular Webpack Starter with AoT compilation, Lazy-loading, and Tree-shaking
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_your_repo **}}**](undefined)
```
Comment (rolandjitsu) [inline README.md]: More `undefined` here an below.

File: README.md [hunk 30]
```diff
@@ -517,33 +517,33 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 #### Traceur Seed Projects
 
 * [GitHub: Angular2 ES6 Seed (Evan Plaice)](https://github.com/evanplaice/angular2-es6-seed/)
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_your_repo **}}**](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2Fangularclass%2Fawesome-angular2%2Fedit%2Fgh-pages%2FREADME.md)
```
Comment (rolandjitsu) [inline README.md]: This should be left alone.

File: README.md [hunk 31]
```diff
@@ -517,33 +517,33 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 
 ---
 
 ### Angular 2 in Babel
 > The compiler for writing next generation JavaScript.
 
 #### Babel General Resources
-* [Babel](https://babeljs.io/) Official Website for Babel
-* [REPL](https://babeljs.io/repl/) Official Babel REPL that runs entirely in your browser
+* [Babel](undefined) Official Website for Babel
```
Comment (rolandjitsu) [inline README.md]: More of those `undefined`.

File: README.md [hunk 32]
```diff
@@ -517,33 +517,33 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
+* [REPL](undefinedrepl/) Official Babel REPL that runs entirely in your browser
 * [Babel Repository (GitHub)](https://github.com/babel/babel) Official GitHub Repo for Babel
 
 #### Babel Angular 2 Online Playground
 * [Plunker: Angular 2 + Babel](http://plnkr.co/edit/PxCzCu?p=preview)
 
 #### Babel Seed Projects
-* [babel-angular2-app](https://github.com/shuhei/babel-angular2-app) A skeleton Angular 2 app built with [Babel](https://babeljs.io/) and [Browserify](http://browserify.org/).
+* [babel-angular2-app](https://github.com/shuhei/babel-angular2-app) A skeleton Angular 2 app built with [Babel](undefined) and [Browserify](http://browserify.org/).
```
Comment (rolandjitsu) [inline README.md]: `undefined`

File: README.md [hunk 33]
```diff
@@ -517,33 +517,33 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 * [angular2-fullstack-starter](https://github.com/jgodi/angular2-fullstack-starter) A full stack skeleton Angular 2 app built with Webpack/Babel.
-* [angular2-es6-starter](https://github.com/blacksonic/angular2-es6-starter) A skeleton Angular 2 ES6 application built with Babel, Webpack, Gulp.
+* [angular2-es6-starter](https://github.com/blacksonic/angular2-babel-esnext-starter) A skeleton Angular 2 ES6 application built with Babel, Webpack, Gulp.
 * [angular2-es-seed](https://github.com/codewareio/angular2-es-seed) A complete, yet simple, starter for Angular 2 using ES2016.
 * [angular2-babel-quickstart](https://github.com/rbnlffl/angular2-babel-quickstart) The super simple Angular 2 quickstarter with ES6/Babel
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_your_repo **}}**](undefined)
```
Comment (rolandjitsu) [inline README.md]: Should be left alone.

File: README.md [hunk 34]
```diff
@@ -517,33 +517,33 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 
 #### Babel Demo, Samples, and Examples
-* [angular2-es6-todomvc](https://github.com/blacksonic/angular2-es6-todomvc) Angular 2 TodoMVC implementation with ES6.
+* [angular2-es6-todomvc](https://github.com/blacksonic/angular2-esnext-todomvc) Angular 2 TodoMVC implementation with ES6.
 * [ng1-ng2-webpack-lazy-uirouter](https://github.com/swimlane/ng1-ng2-webpack-lazy-uirouter) Hybrid lazy-loading Angular1 + Angular2 using UI-Router, Webpack2 and Babel.
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_your_repo **}}**](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2Fangularclass%2Fawesome-angular2%2Fedit%2Fgh-pages%2FREADME.md)
```
Comment (rolandjitsu) [inline README.md]: This too should be left alone.

File: README.md [hunk 35]
```diff
@@ -555,22 +555,22 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 > An ECMAScript language that includes structured, dynamic, functional, and prototype-based features.
 
 ##### ES5 General Resources
-* [**{{** help_add_resources **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** help_add_resources **}}**](undefined)
```
Comment (rolandjitsu) [inline README.md]: Should not be changed.

File: README.md [hunk 36]
```diff
@@ -555,22 +555,22 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 
 #### ES5 Seed Projects
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_your_repo **}}**](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2Fangularclass%2Fawesome-angular2%2Fedit%2Fgh-pages%2FREADME.md)
```
Comment (rolandjitsu) [inline README.md]: Should not be changed.

File: README.md [hunk 37]
```diff
@@ -555,22 +555,22 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 > Build Realtime Web and Mobile Apps With Angular and Meteor
 
 ##### Meteor General Resources
-* [Angular Meteor](http://www.angular-meteor.com) Official Website for Angular Meteor
+* [Angular Meteor](https://www.angular-meteor.com/) Official Website for Angular Meteor
 * [Angular 2 Meteor](http://angular-meteor.com/angular2)
 
 #### Meteor Seed Projects
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_your_repo **}}**](undefined)
```
Comment (rolandjitsu) [inline README.md]: Should not be changed.

File: README.md [hunk 38]
```diff
@@ -583,8 +583,8 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 
 #### NativeScript Seed Projects
 * [sample-Angular2](https://github.com/NativeScript/sample-Angular2)
-* [angular2-seed-advanced](https://github.com/NathanWalker/angular2-seed-advanced)
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [angular2-seed-advanced](https://github.com/NathanWalker/angular-seed-advanced)
+* [**{{** add_your_repo **}}**](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2Fangularclass%2Fawesome-angular2%2Fedit%2Fgh-pages%2FREADME.md)
```
Comment (rolandjitsu) [inline README.md]: Should not be changed.

File: README.md [hunk 39]
```diff
@@ -595,10 +595,10 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 * [React Native](https://facebook.github.io/react-native/) Official Website for React Native
 
 #### React Native Projects
-* [Angular 2 React Native Renderer (GitHub)](https://github.com/angular/react-native-renderer)
+* [Angular 2 React Native Renderer (GitHub)](undefined)
```
Comment (rolandjitsu) [inline README.md]: More `undefined`

File: README.md [hunk 40]
```diff
@@ -595,10 +595,10 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 
 #### React Native Seed Projects
-* [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
+* [**{{** add_your_repo **}}**](undefined)
```
Comment (rolandjitsu) [inline README.md]: Should not be changed.

File: README.md [hunk 41]
```diff
@@ -608,17 +608,17 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 
 ##### Haxe General Resources
 * [Haxe.org](http://haxe.org) 
-* [Haxe on Wikipedia](https://en.wikipedia.org/wiki/Haxe)
+* [Haxe on Wikipedia]()
```
Comment (rolandjitsu) [inline README.md]: The url is gone.

File: README.md [hunk 42]
```diff
@@ -608,17 +608,17 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
...
 * [Haxe Playground ](http://try.haxe.org/)
 * [Haxe compiler targets](http://haxe.org/documentation/introduction/compiler-targets.html) compiler targets: C++, C#, Java, Python, PHP, ActionScript 3, and Flash or Neko bytecode
 * [Haxe Usage Survey results](blog.onthewings.net/2015/11/14/haxe_usage_survey/)
-* [Haxe discussion group](http://groups.google.com/group/haxelang) - community 
+* [Haxe discussion group](https://groups.google.com/forum/#!forum/haxelang) - community 
 * [Haxe and Node.js](matthijskamstra.github.io/haxenode/) 
-* [Haxe Resource](http://haxeresource.meteor.com/) - some "hub" for articles
+* [Haxe Resource]() - some "hub" for articles
```
Comment (rolandjitsu) [inline README.md]: Url is gone.

File: README.md [hunk 43]
```diff
@@ -641,7 +641,7 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 #### Security
 * [Angular 2 with OpenID Connect Implicit Flow from Damien Bowden](https://damienbod.com/2016/03/02/angular2-openid-connect-implicit-flow-with-identityserver4/)
 * [Angular 2 bootstrap4 OAuth2 Webpack from Michael Oryl](https://github.com/michaeloryl/angular2-bootstrap4-oauth2-webpack)
-* [Angular 2 OAuth2 OIDC from Manfred Steyer](http://www.softwarearchitekt.at/post/2016/07/03/authentication-in-angular-2-with-oauth2-oidc-and-guards-for-the-newest-new-router-english-version.aspx)
+* [Angular 2 OAuth2 OIDC from Manfred Steyer]()
```
Comment (rolandjitsu) [inline README.md]: The url is gone.

**Comment by rolandjitsu** [comment]
@vlio20 I took care of all broken urls. Please make another PR with only your own additions. Thanks though for the help 👍

**Pull request closed without merging**